proto/v1alpha1/rode.proto

```diff
@@ -103,6 +103,16 @@ service Rode {
     };
     option (google.api.method_signature) = "id,update_mask";
   }
+  // RegisterCollector accepts a collector ID and a list of notes that this collector will reference when creating
+  // occurrences. The response will contain the notes with the fully qualified note name. This operation is idempotent,
+  // so any notes that already exist will not be re-created. Collectors are expected to invoke this RPC each time they
+  // start.
+  rpc RegisterCollector(RegisterCollectorRequest) returns (RegisterCollectorResponse) {
+    option (google.api.http) = {
+      post: "/v1alpha1/collectors:register"
```

Comment:
If this endpoint operates on a single collector  using its id, should the path be `/v1alpha1/collectors/{id}:register`?